src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -1243,10 +1243,12 @@
 			else
 				value = "EditAndContinue"
 			end
+		elseif cfg.symbols == p.OFF then
+			value = ""
```

Comment:
#541 fixes `symbols` it more properly. This PR should be split so the different fixes can be merged independently.